os2forms_dig_sig_server.services.yml

```
@@ -0,0 +1,13 @@
+services:
+  logger.channel.os2forms_dig_sig_server:
+    parent: logger.channel_base
+    arguments: [ 'os2forms_dig_sig_server' ]
+  os2forms_dig_sig_server.signing_service:
+    class: Drupal\os2forms_dig_sig_server\Service\SigningService
```

Consider using the FQCN as service name (cf. https://www.drupal.org/docs/drupal-apis/services-and-dependency-injection/services-and-dependency-injection-in-drupal#s-defining-services-by-fully-qualified-name-of-php-namespace):

```suggestion
  Drupal\os2forms_dig_sig_server\Service\SigningService:
```